rdt/hyper_transformer.py

```diff
@@ -381,7 +380,7 @@ def update_sdtypes(self, column_name_to_sdtype):
                     supported_sdtypes = current_transformer.get_supported_sdtypes()
 
                 if sdtype not in supported_sdtypes:
-                    transformers_to_update[column] = get_default_transformer(sdtype)
+                    transformers_to_update[column] = deepcopy(get_default_transformer(sdtype))
```

Comment:
get_default_transformer caches the values, so it always returns  references to the same instances, so we gotta copy them.